ci-operator/step-registry/openshift-qe/installer/featuregate/openshift-qe-installer-featuregate-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+oc patch  featuregate cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' --type merge
+
+sleep 300
```

Comment:
Will this sleep/wait be sufficient at large-scale clusters like 120/249 nodes?

Reply:
This wait is just for the task of restarts to begin, the loop below should wait another hour if needed.